myhpom/views/signup.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import render, redirect
+from django.contrib import auth
+from myhpom.forms.signup import SignupForm
+from myhpom.models import User, UserDetails, State
+
+
+def signup(request):
+    """
+    TODO:
```

Comment:
one TODO looks to be addressed already, so that comment could go (email already taken)